src/language/dafnyLanguageClient.ts

```diff
@@ -61,9 +76,14 @@ export class DafnyLanguageClient extends LanguageClient {
     return this.sendRequest<ICounterExampleItem[]>('dafny/counterExample', param);
   }
 
-  public static async create(context: ExtensionContext): Promise<DafnyLanguageClient> {
+  public static async create(context: ExtensionContext, statusOutput: OutputChannel): Promise<DafnyLanguageClient> {
     const dotnetExecutable = await getDotnetExecutablePath();
     const launchArguments = [ getLanguageServerRuntimePath(context), ...getLanguageServerLaunchArgs() ];
+    statusOutput.appendLine(`Language server arguments: ${launchArguments.map(oneArgument =>
```

Comment:
Please extract the "prettifying" of the arguments into a separate function for readability. The best would be to extract the whole part that's responsible to turn the `launchArguments` array into a string.